src/main/java/de/retest/util/ChecksumCalculator.java

```diff
@@ -18,15 +17,12 @@ public static ChecksumCalculator getInstance() {
 		return instance;
 	}
 
-	private final HashFunction sha256 = Hashing.sha256();
-	private final HashFunction md5 = Hashing.md5();
-
 	public String sha256( final String input ) {
-		return sha256.hashString( input, Charset.defaultCharset() ).toString();
+		return getDigest( "SHA-256", input.getBytes( StandardCharsets.UTF_8 ) );
```

Comment:
Aren't there constants for these strings?

Reply:
I don't think so.